pingora-core/src/server/configuration/mod.rs

```diff
@@ -61,11 +62,11 @@ pub struct ServerConf {
     /// defined by the SSL library will be used.
     pub ca_file: Option<String>,
     // These options don't belong here as they are specific to certain services
-    pub(crate) client_bind_to_ipv4: Vec<String>,
-    pub(crate) client_bind_to_ipv6: Vec<String>,
-    pub(crate) upstream_keepalive_pool_size: usize,
-    pub(crate) upstream_connect_offload_threadpools: Option<usize>,
-    pub(crate) upstream_connect_offload_thread_per_pool: Option<usize>,
+    pub client_bind_to_ipv4: Vec<String>,
```

Comment:
These fields are essentially unstable right now as we're thinking about moving or renaming these fields elsewhere. We can add comments to indicate that.